src/OrbitGl/CallstackThreadBar.cpp

```diff
@@ -85,7 +88,7 @@ void CallstackThreadBar::DoDraw(PrimitiveAssembler& primitive_assembler,
   primitive_assembler.AddLine(pos, Vec2(x1, y0), event_bar_z, color, shared_from_this());
   primitive_assembler.AddLine(Vec2(x1, y1), Vec2(x0, y1), event_bar_z, color, shared_from_this());
 
-  if (picked_) {
+  if (picked_ && !absl::GetFlag(FLAGS_time_range_selection)) {
```

Comment:
1) Nit: I know that this was here before, but I would prefer an early return:

`if (!picked_ || absl::GetFlag(FLAGS_time_range_selection)) return;`

2) Please add a comment to explain why we are using this anymore (because it's replaced by time range selection)